lib/cisco_node_utils/cmd_ref/interface_channel_group.yaml

```diff
@@ -16,18 +16,34 @@ channel_group:
   kind: int
   default_value: false
   nexus:
-    get_value: '/^channel-group (\d+)$/'
+    get_value: '/^channel-group (\d+)\s?\S*\s?\S*$/'
```

Comment:
We don't need to include this change with the current design.  I was only suggesting an alternative way of capturing both the `id` and `mode` but since you have the `channel_group_mode` getter below this can remain unchanged.

`get_value: '/^channel-group (\d+)$/'`